### ISSUE TITLE
Kotlin cant find source apk (with solution)

### ISSUE THREAD

**Issue by maxiking913** [ReVanced/revanced-manager#1071]
### Type

Error while running the manager

### Bug description

Cant patch YouTube. No Matter which Patch. (Tried one by one)

### Steps to reproduce

Install and Patch youtube

### Android version

Version 13

### Manager version

1.4.3

### Target package name

youtube

### Target package version.

18.23.35

### Installation type

Doesnt Matter (doesnt come to this Point) but rooted

### Patches selected.

Doesnt matter

### Device logs (exported using Manager settings).

```shell
--------- beginning of system
08-02 10:09:06.996  8904  8904 E MQSEventManagerDelegate: failed to get MQSService.
08-02 10:09:06.996  8904  8904 W MQSEventManagerDelegate: registerApplicationScoutThread getMQSService null
08-02 10:09:06.996  8904  8904 W ActivityThread: registerApplicationScoutThread result:false
08-02 10:09:07.138  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
08-02 10:09:10.034  8904  8926 W MIUIScout App: Enter APP_SCOUT_WARNING State
08-02 10:09:10.035  8904  8926 W MIUIScout App:  (Current message: duration=2504ms seq=43 late=26ms h=android.os.Handler c=io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0)
08-02 10:09:10.037  8904  8926 E MQSEventManagerDelegate: failed to get MQSService.
08-02 10:09:10.037  8904  8926 W MQSEventManagerDelegate: registerApplicationScoutThread getMQSService null
08-02 10:09:10.037  8904  8926 W ActivityThread: registerApplicationScoutThread result:false
08-02 10:09:10.038  8904  9120 W MIUIScout App: Event:APP_SCOUT_WARNING Thread:main backtrace:
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at java.lang.Object.wait(Native Method)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at java.lang.Object.wait(Object.java:442)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at java.lang.Thread.join(Thread.java:1531)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at com.stericson.RootTools.execution.Shell.<init>(Shell.java:85)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:513)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:495)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at com.stericson.RootTools.internal.RootToolsInternalMethods.isAccessGiven(RootToolsInternalMethods.java:764)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at com.stericson.RootTools.RootTools.isAccessGiven(RootTools.java:538)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at com.gokul.root.RootPlugin.isRooted(RootPlugin.java:51)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at com.gokul.root.RootPlugin.onMethodCall(RootPlugin.java:41)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:258)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:322)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at android.os.Handler.handleCallback(Unknown Source:2)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at android.os.Handler.dispatchMessage(Unknown Source:4)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at android.os.Looper.loopOnce(Unknown Source:182)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at android.os.Looper.loop(Unknown Source:82)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at android.app.ActivityThread.main(Unknown Source:108)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at java.lang.reflect.Method.invoke(Native Method)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(Unknown Source:11)
08-02 10:09:10.038  8904  9120 W MIUIScout App:     at com.android.internal.os.ZygoteInit.main(Unknown Source:312)
08-02 10:09:10.038  8904  8926 E MQSEventManagerDelegate: failed to get MQSService.
08-02 10:09:10.038  8904  8926 W MQSEventManagerDelegate: reportAppScoutEvent getMQSService null
08-02 10:09:10.244  8904  8904 W Looper  : PerfMonitor longMsg : seq=43 plan=10:09:07.504 late=26ms wall=2709ms running=2ms h=android.os.Handler c=io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0 procState=-1
08-02 10:09:10.248  8904  8904 W Looper  : PerfMonitor doFrame : time=1ms vsyncFrame=0 latency=2739ms procState=-1 historyMsgCount=7 (msgIndex=7 wall=2709ms seq=43 running=2ms late=26ms h=android.os.Handler c=io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0)
08-02 10:09:10.325  8904  8904 W Looper  : PerfMonitor looperActivity : package=app.revanced.manager.flutter/.MainActivity time=1ms latency=2566ms running=0ms  procState=-1  historyMsgCount=9 (msgIndex=1 wall=2709ms seq=43 running=2ms late=26ms h=android.os.Handler c=io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0) (msgIndex=4 wall=60ms seq=46 running=4ms runnable=1ms late=2729ms h=android.os.Handler c=io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0)
08-02 10:09:10.326  8904  8904 W Looper  : PerfMonitor looperActivity : package=app.revanced.manager.flutter/.MainActivity time=2ms latency=2566ms running=1ms  procState=-1  historyMsgCount=10 (msgIndex=1 wall=2709ms seq=43 running=2ms late=26ms h=android.os.Handler c=io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0) (msgIndex=4 wall=60ms seq=46 running=4ms runnable=1ms late=2729ms h=android.os.Handler c=io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0)
08-02 10:09:16.722  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
08-02 10:09:20.533  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
08-02 10:11:27.338  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
08-02 10:11:27.694  8904  8904 W PackageParser: Unknown element under <manifest>: queries at /data/user/0/app.revanced.manager.flutter/cache/file_picker/com.google.android.youtube_18.23.35-1538252224_minAPI26(arm64-v8a,armeabi-v7a,x86,x86_64)(nodpi)_apkmirror.com.apk Binary XML file line #41
08-02 10:11:27.696  8904  8904 W PackageParser: Unknown element under <application>: property at /data/user/0/app.revanced.manager.flutter/cache/file_picker/com.google.android.youtube_18.23.35-1538252224_minAPI26(arm64-v8a,armeabi-v7a,x86,x86_64)(nodpi)_apkmirror.com.apk Binary XML file line #417
08-02 10:11:59.105  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
08-02 10:12:55.088  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
08-02 10:14:01.268  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
08-02 10:14:38.883  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
--------- beginning of main
08-02 10:15:25.369  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:15:25.369  8904  8904 I HandWritingStubImpl: refreshLastKeyboardType: 1
08-02 10:15:25.369  8904  8904 I HandWritingStubImpl: getCurrentKeyboardType: 1
08-02 10:15:25.371  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:15:25.375  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:15:57.530  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
08-02 10:15:57.555  8904  8925 D AppScoutStateMachine: 8904-ScoutStateMachinecreated
08-02 10:15:57.571  8904  8904 D SurfaceView: UPDATE Surface(name=SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity])/@0x83372e7, mIsProjectionMode = false
08-02 10:15:57.571  8904  8904 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-02 10:15:57.571  8904  8904 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-02 10:15:57.571  8904  8904 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=0
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=1
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=2
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=3
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=4
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=5
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=6
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=7
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=8
08-02 10:15:57.571  8904  8904 E SurfaceSyncer: Failed to find sync for id=9
08-02 10:15:57.599  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:15:57.599  8904  8904 I HandWritingStubImpl: refreshLastKeyboardType: 1
08-02 10:15:57.599  8904  8904 I HandWritingStubImpl: getCurrentKeyboardType: 1
08-02 10:15:57.602  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:15:57.602  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:15:57.604  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:16:02.266  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:16:02.266  8904  8904 I HandWritingStubImpl: refreshLastKeyboardType: 1
08-02 10:16:02.266  8904  8904 I HandWritingStubImpl: getCurrentKeyboardType: 1
08-02 10:16:02.268  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:16:02.278  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:16:02.874  8904  8904 E BLASTBufferQueue: [VRI[MainActivity]#16](f:0,a:2) Applying pending transactions on dtor 1
08-02 10:16:07.544  8904  8904 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for app.revanced.manager.flutter.MainActivity@a594f68, reason: AppDarkModeEnable
08-02 10:16:07.561  8904 10257 D AppScoutStateMachine: 8904-ScoutStateMachinecreated
08-02 10:16:07.571  8904  8904 D SurfaceView: UPDATE Surface(name=SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity])/@0xddb8b2e, mIsProjectionMode = false
08-02 10:16:07.571  8904  8904 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-02 10:16:07.571  8904  8904 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-02 10:16:07.571  8904  8904 D SurfaceControl: nativeSetScreenProjection ===> setScreenProjection
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=0
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=1
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=2
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=3
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=4
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=5
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=6
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=7
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=8
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=9
08-02 10:16:07.572  8904  8904 E SurfaceSyncer: Failed to find sync for id=10
08-02 10:16:07.617  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:16:07.617  8904  8904 I HandWritingStubImpl: refreshLastKeyboardType: 1
08-02 10:16:07.617  8904  8904 I HandWritingStubImpl: getCurrentKeyboardType: 1
08-02 10:16:07.620  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:16:07.620  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
08-02 10:16:07.622  8904  8904 I HandWritingStubImpl: isKeyboardTypeChanged: false
```

### Installer logs (exported using Installer menu option) [unneeded if the issue is not during patching].

```shell
Initializing installer
Creating working directory
Copying original apk
An error occurred! Aborted
Error:
kotlin.io.NoSuchFileException: /data/adb/revanced/com.google.android.youtube/original.apk: The source file doesn't exist.
	at kotlin.io.FilesKt__UtilsKt.copyTo(Utils.kt:219)
	at kotlin.io.FilesKt__UtilsKt.copyTo$default(Utils.kt:217)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda-20(MainActivity.kt:114)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$Rd-IpIH_jmxD15EWmBEFM-jtGsk(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda13.run(Unknown Source:24)
	at java.lang.Thread.run(Thread.java:1012)
```

### Screenshots or video

No need its the Same AS in logs

### Solution

Remove original and Base apk Out of the given path with root Explorer.

Maybe remove them If exist b4 each Patch (possible fix)

### Acknowledgments

- [ ] I have searched the existing issues; this is new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I properly filled out all of the requested information in this issue.
- [X] The issue is solely related to ReVanced Manager and not caused by patches.

**Comment by reisxd** [comment]
Acknowledgements not met.

**Comment by maxiking913** [comment]
> Duplicate of #975

Solution included

**Comment by TheAabedKhan** [comment]
Fixed in #1052, please wait for the next update
